### PR TITLE
Msopof contrib microsoft defender advanced threat protection - machine offboard

### PR DESCRIPTION
**Pull request by msopof**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by msopof** [comment]
@israelpoli Hi, can you look at https://app.circleci.com/pipelines/github/demisto/content/176385/workflows/741121a9-e4da-48ff-903c-882af82d4294/jobs/542105

And tell what is exactly wrong with file and how to fix it?

**Comment by msopof** [comment]
Any update regarding the secret test fail?

File: Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/MicrosoftDefenderAdvancedThreatProtection.py [hunk 1]
```diff
@@ -1983,6 +1997,43 @@ def get_user_data(user_response):
     return user_data
 
 
+def offboard_machine_command(client: MsClient, args: dict):
+    """Offboard machine from defender.
+
+    Returns:
+        (str, dict, dict). Human readable, context, raw response
+    """
+    headers = ['ID', 'Type', 'Requestor', 'RequestorComment', 'Status', 'MachineID', 'ComputerDNSName']
```
Comment (israelpoli) [inline Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/MicrosoftDefenderAdvancedThreatProtection.py:2006]: Please add a validate if the arguments were indeed provided

```suggestion
    if not args.get('machine_id') or not args.get('comment'):
        raise ValueError("something")
    headers = ['ID', 'Type', 'Requestor', 'RequestorComment', 'Status', 'MachineID', 'ComputerDNSName']
```

File: Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/README.md [hunk 2]
```diff
@@ -5843,6 +5844,77 @@ Alert.ReadWrite.All
...
+| MicrosoftATP.MachineAction.Requestor | String | The ID of the user that executed the action.               | 
+| MicrosoftATP.MachineAction.RequestorComment | String | Comment that was written when issuing the action.          | 
+| MicrosoftATP.MachineAction.Status | String | The current status of the command.                         | 
+| MicrosoftATP.MachineAction.MachineID | String | The machine ID on which the action was executed.           | 
+| MicrosoftATP.MachineAction.ComputerDNSName | String | The machine DNS name on which the action was executed.     | 
+| MicrosoftATP.MachineAction.CreationDateTimeUtc | Date | The date and time when the action was created.             | 
+| MicrosoftATP.MachineAction.LastUpdateTimeUtc | Date | The last date and time when the action status was updated. | 
+| MicrosoftATP.MachineAction.cancellationDateTimeUtc | Date | The date and time when the action was canceled.      |
+| MicrosoftATP.MachineAction.RelatedFileInfo | String | The file info.                                             | 
+| MicrosoftATP.MachineAction.troubleshootInfo | String | Troubleshooting information.                               |
```
Comment (israelpoli) [inline Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/README.md:5881]: After you change the keys of the outputs in the `yml` file, please also change the `REDME` file

File: Packs/MicrosoftDefenderAdvancedThreatProtection/ReleaseNotes/1_11_6.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+##### Microsoft Defender for Endpoint
+-  1.9.1
```
Comment (israelpoli) [inline Packs/MicrosoftDefenderAdvancedThreatProtection/ReleaseNotes/1_11_6.md:4]: Please remove this line.

File: Packs/MicrosoftDefenderAdvancedThreatProtection/ReleaseNotes/1_9_0.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
```
Comment (israelpoli) [inline Packs/MicrosoftDefenderAdvancedThreatProtection/ReleaseNotes/1_9_0.md:1]: Please restore the file to its previous state

**Review by israelpoli — CHANGES_REQUESTED** [review]
Hi @msopof...
Thank you for your contributions.

**Note** -> you have a `content` folder at branch, please delete it.
Please see my comments below.

Thanks

**Comment by msopof** [comment]
@israelpoli Finally all test passed. Can you approve changes?

**Comment by israelpoli** [comment]
Hi @msopof 

Please run:
1. `git checkout origin/master ./Packs/MicrosoftDefenderAdvancedThreatProtection/ReleaseNotes/1_9_0.md`
2. `git checkout origin/master ./Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/Pipfile`
3. `git checkout origin/master ./Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/Pipfile.lock`


**Commit and Push**

**Comment by msopof** [comment]
@israelpoli Done.

**Review by israelpoli — APPROVED** [review]
Thanks for your contribution.

Good job

**Pull request merged**